Model/Sales/MagentoOrderCollection.php

```diff
@@ -222,14 +222,23 @@ public function setFulfilment(): self
             $this->setBillingRecipient();
             $this->setShippingRecipient();
 
+            $dropOffPoint = $this->helper->getDropOffPoint(
+                CarrierFactory::createFromName($deliveryOptionsAdapter->getCarrier())
+            );
+
+            if (! $dropOffPoint && CarrierInstabox::NAME === $deliveryOptionsAdapter->getCarrier()) {
```

Comment:
Of de dropoffpoint aanwezig moet zijn zit toch in de sdk dacht ik, had je dat niet aangepast ihkv deze story? Ik zou niet zo'n check in de plugin willen hebben, met de naam van de carrier zo expliciet vermeld.
Wanneer de sdk een error throwt bij setDropOffPoint dan zou je die kunnen afvangen verderop.